diff --git a/node_modules/@clerk/shared/dist/runtime/loadClerkJsScript.mjs b/node_modules/@clerk/shared/dist/runtime/loadClerkJsScript.mjs
index 03ac56b..0cf152f 100644
--- a/node_modules/@clerk/shared/dist/runtime/loadClerkJsScript.mjs
+++ b/node_modules/@clerk/shared/dist/runtime/loadClerkJsScript.mjs
@@ -11,4 +11,4 @@ import "./url-Cdy8w8vK.mjs";
 import "./versionSelector-C_XZNCEb.mjs";
 import { buildClerkJsScriptAttributes, clerkJsScriptUrl, loadClerkJsScript, setClerkJsLoadingErrorPackageName } from "./loadClerkJsScript-BtksLb-o.mjs";
 
-export { buildClerkJsScriptAttributes, clerkJsScriptUrl, loadClerkJsScript, setClerkJsLoadingErrorPackageName };
\ No newline at end of file
+export { buildClerkJsScriptAttributes, clerkJsScriptUrl, loadClerkJsScript, setClerkJsLoadingErrorPackageName };export { loadClerkJsScript as loadClerkUiScript } from "./loadClerkJsScript-BtksLb-o.mjs"; 
